task1.2/task1.2.c

```diff
@@ -0,0 +1,41 @@
+﻿#include <stdio.h>
```

Comment:
#define _USE_MATH_DEFINES // for C
#include <math.h>
https://learn.microsoft.com/ru-ru/cpp/c-runtime-library/math-constants?view=msvc-170